src/objective/objective_function.cpp

```diff
@@ -39,7 +40,7 @@ ObjectiveFunction* ObjectiveFunction::CreateObjectiveFunction(const std::string&
       return new CUDABinaryLogloss(config);
     } else if (type == std::string("lambdarank")) {
       Log::Warning("Objective lambdarank is not implemented in cuda_exp version. Fall back to boosting on CPU.");
```

Comment:
remove this warning?

Reply:
Done via [573bfd5](https://github.com/microsoft/LightGBM/pull/5453/commits/573bfd567cf3765a3bf5bd9e85f3cd32840b472c)